README.md

```diff
@@ -4,39 +4,41 @@
 
 # matrix-registration
 
-a simple python application to have a token based matrix registration
+A simple Python application enabling token-based registration for matrix servers.
 
-if you like me encountered the situation where you want to invite your friends to your homeserver, but neither wanted to open up public registration nor create accounts for every individual user yourself, this project should be the solution.
+You may have, like me, encountered the situation where you want to invite your friends to create an account on your homeserver, but neither want to open up public registration nor create accounts for every individual user yourself. This project aims to solve this problem.
 
-with this project you can just quickly generate tokens on the fly and share them with your friends to allow them to register to your homeserver.
+With matrix-registration, you can quickly generate tokens on the fly and share them with your friends to allow them to register on your homeserver.
 
 <img src="https://matrix.org/_matrix/media/v1/download/dmnd.sh/UKGgpbHRdFXzKywxjjbfHAsI" width="500">
 
 
 ## setup
+Install using pip:
 
 ```bash
 pip3 install matrix-registration
 matrix-registration
 ```
+
+### First start
+Generate a configuration for your matrix homeserver by copying [config.sample.yaml](/config.sample.yaml) to your server and editing it:
+```bash
+wget https://raw.githubusercontent.com/ZerataX/matrix-registration/master/config.sample.yaml
+cp config.sample.yaml config.yaml
+nano config.yaml
+```
+
+Then pass the path to this configuration to the application on startup using `--confiig-path /path/to/config.yaml`.
```

Comment:
usually unnecessary if you just follow the actual setup procedure

Reply:
Hello,

There is an error it's --config-path [path/to/config.yaml] above.

I think it should be added to the wiki too. I just installed the script today and it was a bit annoying that the wiki do not talk where to modify or get and example of the config.yaml since you said : "You should definitely take a look at the actual configuration file."

Regards

Reply:
well it tells you where it is after you executed `matrix-registration` for the first time